documentation/man/kw.rst

```diff
@@ -17,14 +17,14 @@ Inglorious kernel developer workflow scripts
 
 SYNOPSIS
 ========
-**kw** *COMMAND* [*OPTION* ...]
+*kw* *<command>* [<option> ...]
 
 DESCRIPTION
 ===========
 **kw** is a set of scripts that have a single mission: reduce the overhead
```

Comment:
Since we are here, let replace `**kw** is a set of scripts that have a single mission` to `**kw** mission is`